server_test.go

```diff
@@ -830,6 +830,195 @@ func TestServer_WriteSeries(t *testing.T) {
 	}
 }
 
+// Ensure the server can drop a measurement.
+func TestServer_DropMeasurement(t *testing.T) {
```

Comment:
Add a unit test to ensure the the server does the right thing if the Measurement doesn't exist.